hw3/indexhw3.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>hw3</title>
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/fonts.css">
+	<link rel="stylesheet" href="css/stylehw3.css">
+	<link rel="stylesheet" href="css/response.css">
+</head>
+<body>
+	<header class="header">
+		<div class="header__wrapper">
+			<span class="header__logo">Global</span>
+			<ul class="header__menu">
+				<li class="header__menu__item"><a href="#" class="header__menu__link">Home</a></li>
+				<li class="header__menu__item"><a href="#portfolio-anchor" class="header__menu__link">Portfolio</a></li>
+				<li class="header__menu__item"><a href="#about-anchor" class="header__menu__link">About</a></li>
+				<li class="header__menu__item"><a href="#services-anchor" class="header__menu__link">Services</a></li>
+				<li class="header__menu__item"><a href="#projects-anchor" class="header__menu__link">Projects</a></li>
+				<li class="header__menu__item"><a href="#contact-anchor" class="header__menu__link">Contact</a></li>
+			</ul>			
+			<div class="responsive-menu">
+				<span>&#9776;</span>
+				<ul class="dropdown-menu">
+					<li class="dropdown-menu__item"><a class="dropdown-menu__link" href="#">Home</a></li>
+					<li class="dropdown-menu__item"><a class="dropdown-menu__link" href="#portfolio-anchor">Portfolio</a></li>
+					<li class="dropdown-menu__item"><a class="dropdown-menu__link" href="#about-anchor">About</a></li>
+					<li class="dropdown-menu__item"><a class="dropdown-menu__link" href="#services-anchor">Services</a></li>
+					<li class="dropdown-menu__item"><a class="dropdown-menu__link" href="#projects-anchor">Projects</a></li>
+					<li class="dropdown-menu__item"><a class="dropdown-menu__link" href="#contact-anchor">Contact</a></li>
+				</ul>
+			</div>
+		</div>
+	</header>
+	<div class="banner">
+		<div class="banner__wrapper">
+			<div class="banner__page1"><img src="img/school-img.png"></div>
+			<div class="banner__page2"><img src="img/kil-img.png"></div>
+			<div class="banner__page3"><img src="img/perpp-img.png"></div>
+			<div class="banner__info">
+				<span class="banner__info__title">
```

Comment:
banner__title is ok as far as it is the only title of the block as well as text and button